packages/plugin-commands-patching/src/patch.ts

```diff
@@ -35,16 +35,21 @@ export function help () {
           description: 'The package that needs to be modified will be extracted to this directory',
           name: '--edit-dir',
         },
+        {
+          description: 'Ignore existing patch files when patching',
+          name: '--ignore-patches',
```

Comment:
There may be only one patch, so it should be singular. I suggest to use `--ignore-existing`

Reply:
Thanks for your suggestion. I'll update it.